libcrux-iot/Cargo.toml

```diff
@@ -12,7 +12,11 @@ repository = "https://github.com/cryspen/libcrux-iot"
 readme = "README.md"
 
 [workspace.dependencies]
-libcrux-secrets = { version = "0.0.3" }
+# XXX: We need the git dependency for two reasons:
+# - https://github.com/cryspen/libcrux/pull/1135, which implements (de-)classification for public slices is not in a released version, yet.
+# - https://github.com/cryspen/libcrux/pull/1210, which makes the impls for `(De)ClassifyRef` visible to hax, is not in a released version, yet.
+libcrux-secrets = { git = "https://github.com/cryspen/libcrux" }
```

Comment:
You can keep the version field and just add `git`. This will happen more often and it doesn't hurt keeping that around. We just need the released version once we release here.